src/components/Header.js

```diff
@@ -0,0 +1,7 @@
+export default function Header() {
+  return (
+    <div className="header-con">
+      <h2>Cities around the world.</h2>
+    </div>
```

Comment:
- [ ] Great job working on all the project but I think it would be great as well if you add here an image to the header to be like the given design since it is very important to stick with the design layout as mentioned in the requirements 🙏🏻✨ so please make it the same 💯💯

- Your
![image](https://github.com/BranBayou/random-cities/assets/58553711/f50ec4d8-4749-4227-bfb6-5f79a90208ec)

- Given
![image](https://github.com/BranBayou/random-cities/assets/58553711/0a742e46-8121-414e-83a8-921e11180d0c)